recipes/restbed/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm
+import os
+import textwrap
+
+required_conan_version = ">=1.52.0"
+
+
+class RestbedConan(ConanFile):
+    name = "restbed"
+    homepage = "https://github.com/Corvusoft/restbed"
+    description = "Corvusoft's Restbed framework brings asynchronous RESTful functionality to C++14 applications."
+    topics = ("restbed", "restful", "server", "client", "json", "http", "ssl", "tls")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "AGPL-3.0-or-later", "CPL"  # Corvusoft Permissive License (CPL)
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "ssl": [True, False],
+        "ipc": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "ssl": True,
+        "ipc": False,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+        if self.settings.os in ("Windows", ):
+            del self.options.ipc
+
+    def validate(self):
+        if self.settings.os in ("Windows", ) and not self.options.shared:
+            # The headers don't allow to avoid __declspec(dllimport)/__declsped(dllexport)
+            raise ConanInvalidConfiguration("Static libraries for Windows are not supported")
```

Comment:
Indeed, it comes from https://github.com/Corvusoft/restbed/pull/434

Reply:
I fixed it with a regex.